### PR TITLE
Updated Block abuse of exploited vulnerable signed drivers

### PR DESCRIPTION
**Pull request by JoelJerkin**
Updated technical accuracy for Block abuse of exploited vulnerable signed drivers since it's not available yet in Intune or Configuration Manager yet.'
#ATCP

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/defender-endpoint/attack-surface-reduction-rules.md [hunk 1]
```diff
@@ -141,8 +141,10 @@ The **Block abuse of exploited vulnerable signed drivers** rule does not block a
 >
 > To have a driver examined, use this Web site to [Submit a driver for analysis](https://www.microsoft.com/en-us/wdsi/driversubmission).
 
-Intune Name: `Block abuse of exploited vulnerable signed drivers`
+Intune Name: `Block abuse of exploited vulnerable signed drivers` (Not yet available)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/attack-surface-reduction-rules.md]:
```suggestion
Intune Name: `Block abuse of exploited vulnerable signed drivers` (not yet available)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by yogkumgit** [comment]
@JoelJerkin Please implement the feedback. Thanks

**Comment by JoelJerkin** [comment]
> @JoelJerkin Please implement the feedback. Thanks

Done!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@jweston-1 Please merge this PR. Thanks.

**Comment by BarbaraSelden** [comment]
@jweston-1 this contribution now appears to be out of date (branch) but still needs to be published. Can you help?

**Comment by yogkumgit** [comment]
@JoelJerkin

**Comment by JoelJerkin** [comment]
@BarbaraSelden @yogkumgit Done!

**Comment by yogkumgit** [comment]
@jweston-1 /@BarbaraSelden Please merge this PR. Thanks

**Comment by BarbaraSelden** [comment]
I dont have rights to this repo which is why i asked @jweston-1 to merge. can this get done today please?

**Comment by jweston-1** [comment]
Merged, per request.

**Pull request merged**